writing-style-guide.md

```diff
@@ -0,0 +1,267 @@
+
+Avoid words such as would, should, might, and maybe, as it does not inspire confidence to use such language.
+
+Example:
+
+* Avoid: When an inbound message arrives, you might receive it, or you might get an error, or something!
+* Better: When an inbound message arrives, the message callback handler is invoked.
+* Avoid: If you click the button, the alert box will probably be displayed.
+* Better: When you click the `Submit` button, the alert box is displayed.
+
+## Use second person
+
+You should use 'you' rather than 'we' when referring to the reader, especially in task-based material such as tutorials.
+
+The reason for generally avoiding 'we' is that it can potentially create confusion - who exactly is 'we'? 
+
+Consider the following, for example:
+
+* Avoid: We now configure the Time To Live (TTL).
+
+Is that something Ably configures on the developer's behalf, or does the developer have to do it, and if so how?
+
+It is always best to be explicit: if the developer needs to do something, tell them directly what they need to do and how.
+
+Examples:
+
+* *Avoid:* We now need to enter an API Key.
+* *Better:* You now need to enter your API Key in the `Create Application` dialog.
+* *Avoid:* We can now click the button to create your Ably account.
+* *Better:* You can now click the button to create your Ably account.
+* *Best:* Click the `Create` button to create your Ably account.
+
+Use Ably rather than 'we' when referring to the company.
+
+Example:
+
+* *Avoid:* We also provide a Ruby SDK.
+* *Better:* Ably also provides a Ruby SDK.
+
+## Avoid Latin phrases and abbreviations
+
+Latin abbreviations can occasionally cause confusion if not used correctly. For example, people can get confused between the meaning of "i.e." and "e.g." (and sometimes use them as equivalent).
+
+Examples:
+
+* Use ‘for example’, instead of 'e.g.'
+* Use 'that is' rather than 'i.e.'
+* Use 'and so on' instead of 'etc.'.
+
+Sometimes writers create variations of Latin abbreviations, such as 'eg', 'e.g', 'eg.' and so on. It's best, for various reasons, to avoid this. When automated spell checkers are run, these variations can stop the doc build.
+
+Avoid Latin phrases such as "quid pro quo", "ad nauseum", "vis-a-vis". They can also be less convenient to translate.
+
+## Avoid slang
+
+Avoid slang as this can slow down translation and be hard for some developers to interpret.
+
+For example:
+
+* Don't use overly dramatic words like 'crash' or 'implode', use 'error'.
+* Use 'launch' or 'start' rather than 'fire up'.
+* Bear in mind the reader's daily language may not be English.
+* The mouse is clicked and the keyboard is pressed. Avoid terms such as 'hit', 'punch', or 'whack' when referring to the keyboard. Avoid: "Now punch in any key to continue."
+* Avoid idiomatic language such as "You're good to go", "We'll run this by you", "You're all done".
+
+## Avoid words that are rarely used
+
+Avoid unusual words that have a more commonly used equivalent.
```

Comment:
... provided this isn't sacrificing accuracy of meaning.

Reply:
Clarification added.